plugins/catalog-backend/src/ingestion/processors/BuiltinKindsEntityProcessor.ts

```diff
@@ -99,16 +101,29 @@ export class BuiltinKindsEntityProcessor implements CatalogProcessor {
       }
       for (const target of [targets].flat()) {
         const targetRef = parseEntityRef(target, context);
+        if (targetRef.kind === undefined) {
+          throw new Error(
+            'Entity reference kind is undefined and has no default',
```

Comment:
```suggestion
            `Entity reference "${target}" did not specify a kind (e.g. starting with "Component:"), and has no default`,
```

we could also give a `fieldName` param to this function or something, so that we can say in the error message which field that had the error, to be extra super-duper-helpful

Reply:
(you don't have to do the fieldName thing, that was just reflection on future extension of this)